code/game/cas_manager/datums/cas_fire_envelope.dm

```diff
@@ -259,7 +259,26 @@
 		mission_error = "Target is off bounds or obstructed."
 		return
 	change_current_loc(target_turf)
-	playsound(target_turf, soundeffect, 70, TRUE, 50)
+	playsound(source = target_turf, soundin = soundeffect, vol = 70, vary = TRUE, sound_range = 50, falloff = 8)
```

Comment:
You only need to use a named argument for the last of these, since the others are in the same order as defined on the proc: (Unless you're doing it for readability in which case this is fine.)
https://github.com/cmss13-devs/cmss13/blob/8a44ffd2f19cc88e863a3121a4b83f0db78b542a/code/game/sound.dm#L38
<hr>

```suggestion
	playsound(target_turf, soundeffect, 70, TRUE, 50, falloff = 8)
```

Reply:
I think I am gona keep it for readability 70 TRUE 50 is not quite self explanitory, but if the CM coding standards say otherwise I can change it